src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -1317,6 +1317,12 @@
     <Compile Include="$(CommonPath)SkipLocalsInit.cs">
       <Link>Common\SkipLocalsInit.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)System\ITimer.cs">
+      <Link>Common\System\ITimer.cs</Link>
```

Comment:
>Common [](http://example.com/codeflow?start=12&length=6)

@stephentoub added the files to the common as I expect we'll use them when implementing the down-level version.